chore: add community meeting banner, remove AI lab banner

cdrage opened the pull request:
chore: add community meeting banner, remove AI lab banner

### What does this PR do?

* Removes the AI banner and assets
* Adds community banner based upon UX/UI

### Screenshot / video of UI

<!-- If this PR is changing UI, please include
screenshots or screencasts showing the difference -->

Mockup:

![image](https://github.com/user-attachments/assets/92ca2332-83ec-4dfd-bf4d-2bcdfb11811f)


Implementation:
![Screenshot 2025-03-10 at 10 20 16 AM](https://github.com/user-attachments/assets/a5d3a7e6-17e7-45ee-bb7f-9b134fa6248f)
![Screenshot 2025-03-10 at 10 20 19 AM](https://github.com/user-attachments/assets/6a7bfdd8-1111-467b-a256-1332564faf1a)




### What issues does this PR fix or reference?

<!-- Include any related issues from Podman Desktop
repository (or from another issue tracker). -->

Closes https://github.com/containers/podman-desktop-internal/issues/405
Closes https://github.com/podman-desktop/podman-desktop/issues/11547

### How to test this PR?

<!-- Please explain steps to verify the functionality,
do not forget to provide unit/component tests -->

Check netlify / website.

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

cdrage commented:
Few changes vs the mockup:
* `first` instead of `First` = fixed capitalization
* `March 27th` instead of `March 27`.
* font size to match rest of podman desktop site. The chosen font size in the mockup was not compatible / super small vs rest of site.

cc @MariaLeonova

cdrage commented on website/src/components/CommunityBanner.tsx:
```diff
@@ -0,0 +1,23 @@
+        the agenda and RSVP
+        <a
+          href="https://github.com/podman-desktop/community/issues/3"
+          target="_blank"
```
added _blank so it opens a new link / new tab, WDYT of the behaviour / it's okay @benoitf

benoitf commented:
on my end:
not sure about smaller screens
![image](https://github.com/user-attachments/assets/8c2bc712-3ac1-413c-8c45-34c47e1f1cf9)


about the wording:
Usually we don't say 'Click here' but people see the Link and then click on the underlined text

non blocking: the text is way too long to read. If I'm interested I should have details on the link anyway

benoitf commented:
maybe the text should be smaller if the screen is reduced so we always have a single line

From "Join us for the first meeting on March 27" to something very long on large screen

cdrage commented:
> on my end:
> 
> not sure about smaller screens
> 
> ![image](https://github.com/user-attachments/assets/8c2bc712-3ac1-413c-8c45-34c47e1f1cf9)
> 
> 
> 
> 
> 
> about the wording:
> 
> Usually we don't say 'Click here' but people see the Link and then click on the underlined text
> 
> 
> 
> non blocking: the text is way too long to read. If I'm interested I should have details on the link anyway

I'm fine either way! Let us know your thoughts @MariaLeonova 

I'm not available next 3 days on PTO, so feel free to edit the commit @benoitf

rujutashinde commented:
@cdrage This is not adding the banner on the community page, just the main/index page. Will you be adding another PR for adding the banner to the community page?

cdrage commented:
@MariaLeonova @rujutashinde @benoitf 

Made some changes:

* Reduced the text
* Sizing of the boxes
* Smaller spacing between sentence + boxes
* Added to community page too

Which makes it a bit friendlier with smaller screens

Updated in the PR.

![Screenshot 2025-03-10 at 10 20 16 AM](https://github.com/user-attachments/assets/971d0cbf-02c9-40b3-9224-275e9dc227a5)
![Screenshot 2025-03-10 at 10 20 19 AM](https://github.com/user-attachments/assets/af0d29b9-b6dc-4705-84dc-432da0f57ecf)

MariaLeonova commented:
@cdrage thank you for making these updates! What size is the font now in px, please?

rujutashinde commented:
> @cdrage thank you for making these updates! What size is the font now in px, please?

@MariaLeonova Charlie is on PTO, i believe it is` px-5`  looking at the code below in the PR
`<div className="w-full flex flex-row justify-center items-center py-2 bg-gradient-to-r from-sky-700 to-purple-700 overflow-hidden px-5"> `

MariaLeonova commented:
@rujutashinde I don't think that's it, looks a little large to me - wondering if it's due to the screen size captured in [this comment](https://github.com/podman-desktop/podman-desktop/pull/11610#issuecomment-2711438136).

@benoitf do you know what it looks like on a normal sized desktop screen? i.e. 1920×1080?
